packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -312,6 +332,11 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
     },
   });
 
+  const triggerButtonElement =
+    type !== DatepickerType.InputOnly ? (
+      <Button icon={<CalendarIcon />} title="Open calendar" iconOnly disabled={props.disabled} />
```

Comment:
Please let's make a slot from `Button`, so it can be customized if needed

Reply:
Discussed 1:1.

Solved through Popup slot.